de/model.jl

```diff
@@ -130,10 +130,10 @@ function costG!(G, pIn, exp_data)
     end
 
     # Regularization
-    @. G[1:6889] += 0.01 * sign(pIn[1:6889])
+    @. G[1:6889] += 0.1 * sign(pIn[1:6889])
     w = reshapeParams(pIn)[1]
-    T₀ = w * w' - I
-    temp = vec(2 / norm(T₀) * w * T₀)
+    T₀ = w' * w - I
+    temp = vec(20 / norm(T₀) * w * T₀)
```

Comment:
Can you separate the multiply by 10 and move it out of the vec function? That’ll help ensure it’s clear that it’s a scaling factor.